spec/asyncapi.md

```diff
@@ -20,13 +20,14 @@ These files can then be used to create utilities, such as documentation, integra
 The file(s) MUST describe the operations an [application](#definitionsApplication) accepts. For instance, consider the following AsyncAPI definition snippet:
 
 ```yaml
-user/signedup:
+userSignedUp:
+  address: "user/signedup"
```

Comment:
Perhaps we want to keep calling this `path` instead of `address`? I would love to know others opinion. I'm fine with `address` anyway.